pkg/quota/evaluator/core/persistent_volume_claims.go

```diff
@@ -116,3 +132,89 @@ func PersistentVolumeClaimConstraintsFunc(required []api.ResourceName, object ru
 	}
 	return fmt.Errorf("must specify %s", strings.Join(missingSet.List(), ","))
 }
+
+// GroupKind that this evaluator tracks
+func (p *pvcEvaluator) GroupKind() schema.GroupKind {
+	return api.Kind("PersistentVolumeClaim")
+}
+
+// Handles returns true if the evalutor should handle the specified operation.
+func (p *pvcEvaluator) Handles(operation admission.Operation) bool {
+	return admission.Create == operation
+}
+
+// Matches returns true if the evaluator matches the specified quota with the provided input item
+func (p *pvcEvaluator) Matches(resourceQuota *api.ResourceQuota, item runtime.Object) (bool, error) {
+	return generic.Matches(resourceQuota, item, p.MatchingResources, generic.MatchesNoScopeFunc)
+}
+
+// MatchingResources takes the input specified list of resources and returns the set of resources it matches.
+func (p *pvcEvaluator) MatchingResources(items []api.ResourceName) []api.ResourceName {
```

Comment:
Just to make sure I'm following the logic correctly.  We're called here with `gold.storage` and `bronze.storage` and we match both.  Later, `Usage` is called, but we only return `bronze.storage`, but there won't be any failure due to no value for `gold.storage`?